lib/payment_calculator/ecf/contract/uplift_payment_calculations.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require "has_di_parameters"
+
+module PaymentCalculator
+  module Ecf
+    module Contract
+      module UpliftPaymentCalculations
+        extend ActiveSupport::Concern
+
+        included do
+          include HasDIParameters
```

Comment:
Do we need to use this?

Reply:
Yes for now at least. It's keeps the 3 different blocks consistent with each other. May be removed in a PR later, but for now it's much harder to change just one.